example/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+const config = require('../config/.eslintrc.js');
```

Comment:
Is this .eslintrc.js file necessary if it's just using the default eslint config in `../config` without extending it? Similar question for the existing `.babelrc.js` file as well.